### DISCUSSION TITLE
Do accounts with all badges exist?

### DISCUSSION THREAD

**Discussion by Jeff-JungJaeho** [community/community#54402] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hmm. I wonder how many GitHub badge types there are in the first place.

**Comment by radar-blip** [comment, marked as answer]
If you are talking about achievement badges, 
here is a list of them all so far: [Achievement Badges](https://github.com/Schweinepriester/github-profile-achievements)

**Reply by Jeff-JungJaeho** [reply]
thx

**Reply by YasinDehfuli** [reply]
I think this is a better course to learn how to get github achievements : 

https://github.com/4xmen/Get-Github-Achievements

**Comment by om3ga6400** [comment]
there is no one with a badges
